rust/sealevel/client/src/warp_route.rs

```diff
@@ -295,6 +358,57 @@ impl RouterDeployer<TokenConfig> for WarpRouteDeployer {
         }
         .with_client(client)
         .send_with_payer();
+
+        // do the token-2022 cli initialization steps here
+        if let TokenType::Synthetic(token_metadata) = &app_config.token_type {
+            let (mint_account, _mint_bump) =
+                Pubkey::find_program_address(hyperlane_token_mint_pda_seeds!(), &program_id);
+            // the metadata initialization should happen here, but this is via the CLI most likely
```

Comment:
```suggestion
            // the metadata initialization using the CLI
```

Reply:
this was just to aid my implementation, will remove all comments like this one